common/utils/domain.ts

```diff
@@ -1,8 +1,5 @@
 export function getDomain(url: string): string {
```

Comment:
nit: let's not export functions unnecessarily

```suggestion
function getDomain(url: string): string {
```